runtime/context.go

```diff
@@ -0,0 +1,30 @@
+package runtime
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/metadata"
```

Comment:
Thank you for your contribution. 
Would you mind fixing some style issues?

Organize imports in groups.
https://code.google.com/p/go-wiki/wiki/CodeReviewComments#Imports